guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -6,14 +6,24 @@ Configure a downstream {ProjectServer} to synchronize repositories from a connec
 .Prerequisites
 * A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.
 * You imported the subscriptions manifest on both the upstream and downstream {ProjectServer}.
-* You enabled the required repositories for the organization on the upstream {ProjectServer}.
-* You have user permissions to create product repositories and organizations on the downstream {ProjectServer}.
+* On the upstream {ProjectServer}, you enabled the required repositories for the organization.
+* The upstream user is an admin or has the following permissions:
+** `view_organizations`
+** `view_products`
+* On the downstream {ProjectServer}, you have imported the SSL certificate of the upstream {ProjectServer} using the contents of `http://_upstream-{foreman-example-com}_/pub/katello-server-ca.crt`.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing SSL Certificates] in the _Content Management Guide_.
+* The downstream user is an admin or has the following permissions:
+** create product repos ???
+** create organizations ???
+** `edit_organizations` (to import the CA certificate)
+** `view_lifecycle_environments` (if you want to use a custom label downstream) -- should this be `edit_*`?
+** `view_content_views` (if you want to use a custom label downstream) -- should this be `edit_*`?
```

Comment:
And these permissions are probably required on the downstream server only.

Reply:
The user account on the upstream server need to have all the mentioned permissions - `view_organizations`, `edit_organizations`, `view_products`, `view_lifecycle_environments` and `view_content_views`. It's necessary to create that account and limit its permissions because it's used only for the https connection from DS to US. 

But I don't think it's necessary to explicitly list all the permissions for user at downstream server. I would just assume that administration is done by someone with admin account.